python/cudf/cudf/_lib/cpp/aggregation.pxd

```diff
@@ -139,6 +143,10 @@ cdef extern from "cudf/aggregation.hpp" namespace "cudf" nogil:
         string user_defined_aggregator,
         data_type output_type) except +
 
+    cdef unique_ptr[T] make_ewma_aggregation[T](
```

Comment:
Order this at the end (or consistently with the ordering of other aggregators in other files).